libvast/src/system/catalog.cpp

```diff
@@ -27,10 +38,13 @@
 #include "vast/system/report.hpp"
 #include "vast/system/status.hpp"
 #include "vast/table_slice.hpp"
+#include "vast/taxonomies.hpp"
 #include "vast/time.hpp"
 
+#include <caf/attach_stream_sink.hpp>
```

Comment:
I don't think we need this one anymore, do we?

Reply:
True, I removed unused `#include`s in https://github.com/tenzir/vast/pull/2745/commits/5cc691ffcf82df9766d5e853ea6209a1ad615413